data/schedule.yml

```diff
@@ -305,12 +345,19 @@ data:
             link: https://in.pycon.org/cfp/2021/proposals/delivering-successful-api-integrations-with-documentation-driven-development~dwpmM/
             track: 3
 
+          - title: No Code Platform powered by Python & Django
+            speaker: Bhagvan Kommadi
+            type: talk
+            link: https://in.pycon.org/cfp/2021/proposals/no-code-platform-powered-by-python-django~eXDQA/
+            track: 4
+
+
       - time: 16:30
         talks:
-          - title: Panel Discussion 2
-            speaker: ''
+          - title: Diversity in Cybersecurity
+            speaker: Vandana Verma, Shira Shamban, Hitesh Dharmdasani, Judy Ngure
             type: panel
-            link: ''
+            link: https://in.pycon.org/cfp/2021/proposals/panel-diversity-in-cybersecurity~erk9E/
```

Comment:
replace it with https://in.pycon.org/blog/2021/diversity-in-cybersecurity-panel-discussion.html